src/TransactionController.ts

```diff
@@ -129,6 +138,48 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
 		});
 	}
 
+	/**
+	 * Normalizes the transaction information from etherscan
+	 * to be compatible with the TransactionMeta interface
+	 *
+	 * @param txMeta - Object containing the transaction information
+	 * @param currentNetworkID - string representing the current network id
+	 * @returns - TransactionMeta
+	 */
+	private normalizeTxFromEtherscan(txMeta: any, currentNetworkID: string): TransactionMeta {
+		const ts = parseInt(txMeta.timeStamp, 10) * 1000;
+		/* istanbul ignore next */
+		const status = txMeta.isError === '0' ? 'confirmed' : 'failed';
+		return {
+			blockNumber: txMeta.blockNumber,
+			id: random({ msecs: ts }),
+			networkID: currentNetworkID,
+			rawTransaction: '',
+			status,
+			time: ts,
+			transaction: {
+				from: txMeta.from,
+				gas: BNToHex(new BN(txMeta.gas)),
+				gasPrice: BNToHex(new BN(txMeta.gasPrice)),
+				nonce: BNToHex(new BN(txMeta.nonce)),
+				to: txMeta.to,
+				value: BNToHex(new BN(txMeta.value))
+			},
+			transactionHash: txMeta.hash
+		};
+	}
+
+	/**
+	 * Gets the etherscan API url based on the current network
+	 *
+	 * @param currentNetworkID - string representing the current network id
+	 * @returns - String representing the etherscan API url for the current network
+	 */
+	private getEtherscanApiUrl(currentNetworkID: string): string {
```

Comment:
Private functions do not need to be documented. This is private API and shouldn't ever be used by anyone downstream, and internal code should be clear enough to be self-documenting.